### PR TITLE
clear logs and added installation of trailpack

### PR DESCRIPTION
**Pull request by konstantinzolotarev**

### CODE REVIEW

**Comment by YannBertrand** [comment]
Seems fine to me. @tjwebb: PR should be done onto the dev branch, right?

**Comment by konstantinzolotarev** [comment]
@YannBertrand normally yes. This PR is required to complete another task and it will be good if it will be pushed to npm today

File: src/app/steps/install.js [hunk 1]
```diff
@@ -4,6 +4,15 @@
  */
 
 export default function () {
+
+  let server = this.answers['web-engine'];
+
+  // Adding server dependency
+  this.npmInstall('trailpack-' + server, {
```
Comment (tjwebb) [inline src/app/steps/install.js]: I'm not sure if we want to commit to this convention quite yet. For example, private npm modules would be in the form `@orgname/trailpack-whatever`.

Suggestions welcome.

Comment (YannBertrand) [inline src/app/steps/install.js]: If one want to use a private npm module as the webserver, it means that they've created it. Is that really a matter if they need to update the generator also? For a first version at least.

**Pull request merged**